test/framework/src/main/java/org/elasticsearch/test/AbstractWireTestCase.java

```diff
@@ -27,6 +27,10 @@
 import java.io.IOException;
 import java.util.Collections;
 
+/**
```

Comment:
Is `Geometry` the only user of this?

Reply:
SQL also uses it.

Reply:
right, I suppose the previous comment answers that. guess its surface area is small, but provides enough utility! thanks